### PR TITLE
Adds new integration [benleb/surepetcare]

### PR DESCRIPTION
**Pull request by benleb**

### CODE REVIEW

**Comment by benleb** [comment]
🙄 
image

**Review by ludeeus — CHANGES_REQUESTED** [review]
CI does not pass

**Comment by benleb** [comment]
https://github.com/home-assistant/wheels-custom-integrations/pull/404

it fails because I had the change the integration domain `surepetcarebeta`  and repo name `surepetcare` do not match? 😂

**Comment by ludeeus** [comment]
repo name does not matter

**Comment by benleb** [comment]
but that of the json file? 😂 I think that would be something I can fix...^^

**Comment by ludeeus** [comment]
HACS (and HA itself) uses the `domain` key of the manifest.json file for integrations.

**Comment by benleb** [comment]
ah okay nice 👍 that means it's the workflow which is buggy ;) https://github.com/benleb/surepetcare/blob/dev/manifest.json

**Comment by ludeeus** [comment]
it's not

**Comment by benleb** [comment]
any news?

**Comment by gjohansson-ST** [comment]
Does this makes sense to get into HACS even if it would comply with CI?
It clearly states it's development, readme is not correct and manifest point to HA documentation.

**Comment by benleb** [comment]
> Does this makes sense to get into HACS even if it would comply with CI?
> It clearly states it's development, readme is not correct and manifest point to HA documentation.
> Why not making a PR to the official integration instead?

you know that I am the developer of the official integration?^^ you know why this exists at all? :D And don't worry, I am working on a proper documentation ✌️ 

tl;dr its really uncool to wait for the HA release cycle to get a bug fixed that affects your pets

**Comment by gjohansson-ST** [comment]
> you know that I am the developer of the official integration?^^ you know why this exists at all? :D And don't worry, I am working on a proper documentation ✌️
> 
> tl;dr its really uncool to wait for the HA release cycle to get a bug fixed that affects your pets

I did not find a PR in HA for this integration so that was my wondering.
Second it might not be bad as a custom rep, just wondering if it makes sense to have it in HACS default as it is in HA official.

**Comment by benleb** [comment]
The "idea" to do it this way was born after the last HA update which broke the integration for many users. The integration interacts with some hardware devices and I do not own all of them and can therefore not really test the integration with them... After that event, the "beta" variant was introduced and is now used by some users I am in contact with via Github and the HA community forums. And of course, the plan is to "backport" new features and things to the official integration - but just after really being tested.

**Comment by ludeeus** [comment]
As discussed in Discord, I'm closing this 👍
